### ISSUE TITLE
-dsannotations-options not using the minimum SCR version

### ISSUE THREAD

**Issue by amitjoy** [bndtools/bnd#2323]
I have a bundle comprising 5 DS SCR components (using SCR annotations) and I need to restrict all of them to use SCR version 1.2 instead of using the default version 1.3 in bnd v3.5.

I cross-checked the build jar artefact and it contains 5 XML files out of which four XML descriptors use SCR version 1.2 and one XML descriptor uses 1.3. I am not using any DS 1.3 specific annotation attributes in the failing DS component. But still bnd generates XML Descriptors with SCR 1.3 namespace.

For your reference, I have added the following in the bnd file.

```
-dsannotations: *
-dsannotations-options: version;minimum=1.2.0
```

Any idea on what I could be doing wrong?

**Comment by amitjoy** [comment]
I just discovered that SCR 1.3.0 is used if there is any service reference having reference cardinality of 0..n and dynamic policy. I also have reference with cardinality 0..1 but it works perfectly, that is, bnd uses scr 1.2 (N.B: I have remove the other reference that uses 0..n). But as soon as I add the the service reference with 0..n back, bnd only uses 1.3.

**Comment by bjhargrave** [comment]
FYI, Usage questions should be asked in the bndtools-users mail list. See <https://github.com/bndtools/bnd#feedback>. Github issues for for defects and feature requests.

If you think you have found a defect, can you please provide a small github repo which demonstrates the problem? Then we can investigate. Thanks.

**Comment by amitjoy** [comment]
@bjhargrave Thanks for the information. will do it now onwards. BTW I believe I found the silly mistake that I had done. In my service component, the bind method accepts a `ServiceReference<Service>` as second argument which I believe is only available in DS SCR 1.3.

**Comment by bjhargrave** [comment]
Great. I will close this issue then.
